integration_tests/src/main/python/delta_lake_delete_test.py

```diff
@@ -110,6 +110,33 @@ def write_func(spark, path):
     assert_gpu_fallback_write(write_func, read_delta_path, data_path,
                               "ExecutedCommandExec", disable_conf)
 
+@delta_lake
+@ignore_order
+@pytest.mark.skipif(not supports_delta_lake_deletion_vectors(), \
+    reason="Deletion vectors new in Delta Lake 2.4 / Apache Spark 3.4")
```

Comment:
This is not 100% accurate.

https://docs.delta.io/latest/delta-deletion-vectors.html

Do we want to distinguish between the different operators being supported? I am fine with 2.4 being the base for support because scan with nothing that can write them is not really a supported feature we can test.